docs/users/installation.md

```diff
@@ -61,12 +61,33 @@ The first error message means the
 [SemanticDB](https://scalameta.org/docs/semanticdb/guide.html) compiler plugin
 is not enabled for this project. The second error says `RemoveUnused` requires
 the Scala compiler option `-Ywarn-unused`. To fix both problems, add the
-following settings to `build.sbt`
+following settings to `build.sbt`. 
+
+- sbt 1.3.x or above
+
+```diff
+ // build.sbt
+ inThisBuild(
+   List(
+     scalaVersion := "@SCALA212@", // @SCALA211@, or @SCALA213@
++    semanticdbEnabled := true, // enable SemanticDB
++    semanticdbVersion := scalafixSemanticdb.revision // use Scalafix compatible version
+   )
+ )
+
+ lazy val myproject = project.settings(
+   scalacOptions ++= List(
++    "-Ywarn-unused-import" // required by `RemoveUnused` rule
+   )
+ )
+```
+
+- all sbt versions
 
 ```diff
  // build.sbt
```

Comment:
```suggestion
```diff
 // build.sbt, for sbt 1.2.x and older
```

Reply:
IMO it would be nicer to have *explicitly sbt 1.3.x compatible* sub-project specific setting example for new comers. So I'd just add another example.